source/customizing_look_and_feel/02_configuring_navigation/navigation.rst

```diff
@@ -1,7 +1,66 @@
-Configuring the navigation menu
-===============================
+Configuring the navigation
+==========================
 
-Once your page content has been fully defined, you might want to *plug* your page in your domain's menu. This can easily
-be done by editing the menu components in the *Domain > Portal > General settings* section.
+The navigation in an Opendatasoft portal starts with the root page, that welcomes users, and is then mainly guided by the navigation header located at the top of any interface in the front office. Both are configurable via the **Look and feel > Navigation** subsection of the back office.
 
-.. image:: images/navigation.png
+
+Choosing a root page
+--------------------
+
+.. image:: images/navigation_root-page.png
+
+The root page is the default page on which a user lands when accessing the portal via its main URL (e.g. ``domainname.opendatasoft.com/``). By default, the root page of the Explore page, which displays the catalog of datasets of the portal. It can be replaced with any other page build within the domain, such as a home page build with HTML & CSS using the Pages section of the back office (see :doc:`Creating content </creating_content/index>`).
```

Comment:
You can remove the trailing '/' in the url.

Reply:
We may want to explain that the root page must be public and that's why other pages are grey in the selection list.